web/src/storage/search.ts

```diff
@@ -20,10 +20,10 @@
  * find current contact information at www.suse.com.
  */
 
-import { apiModel } from "~/api/storage";
-import { model } from "~/types/storage";
-import { copyApiModel, findDevice, findDeviceIndex } from "~/helpers/storage/api-model";
-import { buildModel } from "~/helpers/storage/model";
+import { model as apiModel } from "~/api/storage";
+import { model } from "~/storage";
+import { build as buildModel } from "~/storage/model";
```

Comment:
to be honest with namespace support as it is in js, I am not sure if it is good idea to have so generic names like build where it is not clear at all what it builds unless you import it under old name. So from my POV I prefer old name, where it was at least clear what it builds ( sadly no longer as there are two storage model with identical name, but it is discussed elsewhere ).